### PR TITLE
c2f11 inputs

### PR DESCRIPTION
**Pull request by Rebecavb**
Inputs for farm 11 cohort 2

### CODE REVIEW

File: input/data/manure_schedule/c2f11_cornsilage_2022_corn_alf_manure_schedule.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
...2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019, 2019],
```
Comment (krb78) [inline input/data/manure_schedule/c2f11_cornsilage_2022_corn_alf_manure_schedule.json:2]: Did Ed end up helping with this or did you just type "2019" 365 times?

**Review by krb78 — APPROVED** [review]
Looks great and runs great :)

**Review by jadamchick — APPROVED** [review]
It seems you're in a good groove on these, Rebeca!

File: input/data/manure_schedule/c2f11_cornsilage_2022_corn_alf_manure_schedule.json (hunk 1 above)
Comment (Rebecavb) [reply input/data/manure_schedule/c2f11_cornsilage_2022_corn_alf_manure_schedule.json:2]: Ed helped me with the code haha

**Pull request merged**